js/adapt-contrib-media.js

```diff
@@ -109,7 +109,17 @@ define([
                 }
             }
 
-            modelOptions.success = _.bind(this.onPlayerReady, this);
+            /*
+                Unless we are on Android/iOS and using native controls, when MediaElementJS initializes the player it will invoke the success callback prior to performing one last call to setPlayerSize. This call to setPlayerSize is deferred by 50ms so we add a delay of 100ms here to ensure that we don't invoke setReadyStatus until the player is definitely finished rendering.
+            */
+
+            var successCallback = this.onPlayerReady.bind(this);
+
+            modelOptions.success = function(mediaElement, domObject) {
+                _.delay(function() {
```

Comment:
rubbish timing I know but just to point out you could do this instead
```suggestion
                _.delay(successCallback, 100, mediaElement, domObject);
```

Reply:
^ just for reference https://underscorejs.org/#delay

Reply:
I'm all for the succinct , thanks :+1: